src/services/rhsm/rhsmTransformers.js

```diff
@@ -149,58 +128,19 @@ const rhsmSubscriptions = (response, { params } = {}) => {
         ...dataResponse
       };
 
-      let normalizedUomValue;
-      if (new RegExp(RHSM_API_PATH_METRIC_TYPES.SOCKETS, 'i').test(uom)) {
-        normalizedUomValue = RHSM_API_PATH_METRIC_TYPES.SOCKETS;
-      }
-
-      if (new RegExp(RHSM_API_PATH_METRIC_TYPES.CORES, 'i').test(uom)) {
-        normalizedUomValue = RHSM_API_PATH_METRIC_TYPES.CORES;
-      }
-
-      updatedData[normalizedUomValue] = totalCapacity;
-      updatedData[SUBSCRIPTIONS_DATA_TYPES.UOM] = normalizedUomValue;
-      updatedData[`hasInfinite${normalizedUomValue}`] = hasInfiniteQuantity;
+      const normalizedMetricId = Object.values(RHSM_API_PATH_METRIC_TYPES).find(value =>
```

Comment:
required: confirm this behavior isn't duplicated in a parallel way under instances transform

Reply:
we went ahead and removed the "normalizedUom" from instances, it wasn't being used.

in addition there were some changes to the way the subscriptions table displays that still require us to "normalize" the data, see sw-2319 and #1297